stable/graylog/values.yaml

```diff
@@ -270,12 +270,22 @@ graylog:
     ##
     # hosts: http://elasticsearch-client.graylog.svc.cluster.local:9200
     hosts: ""
+    # Allow elasticsearch hosts to be fetched from a k8s secret
+    secret:
+      enable: false
+      secretName: ""
+      secretKey: ""
```

Comment:
* Using `secret.enable` is too complicated. It should be tested for existing of secret name variable.
* `graylog.elasticsearch.secret.secretName` can lead to missing understand of the value. I suggest to change the value to `graylog.elasticsearch.uriSecretName`. Also the `graylog.elasticsearch.secret.secretKey` should be `graylog.elasticsearch.uriSecretKey`

Reply:
@KongZ Thanks for the advice. I updated the PR to utilize the variables you mentioned.
But, I check if `uriSecretKey` exists and not `uriSecretName`, because I'm setting `uriSecretName` to `{{ graylog.fullname }}` if the user doesn't provide it as you can see [here](https://github.com/helm/charts/pull/20807/files#diff-4791ca049dfe358d0c8d82394177f47fR109).